File: write_i18n.dart
import 'dart:io';
import 'dart:convert';
void main() {
  final path = r'C:\Users\claud\OneDrive\Desktop\funparks_ids_set_to_com_funparks_app\assets\i18n\tokyo_disneyland.json';
  final f = File(path);
  // Read the existing mirabilandia.json as a template check
  print('Dart is working! Path: $path');
  print('File exists: ${f.existsSync()}');
}

File: write_i18n.js
const fs = require('fs');
const d = JSON.parse(fs.readFileSync('tokyo_i18n_source.json', 'utf8'));
fs.writeFileSync('assets/i18n/tokyo_disneyland.json', JSON.stringify(d, null, 2), 'utf8');
console.log('Done! Size:', fs.statSync('assets/i18n/tokyo_disneyland.json').size, 'bytes');